src/app/containers/Header/index.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import * as svgs from '@bbc/psammead-assets/svgs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import HeaderContainer from '.';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import services from '../../lib/config/services';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+
+storiesOf('Header Container', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        const serviceContextStub = {
+          product: 'BBC News',
+          service,
+          brandSVG: svgs[service],
+          svgHeight: 24,
```

Comment:
I dont think these 3 values need to be in the service context stub, they arnt in the real service configs, the values are inside the brand container